common/ethnicities/wc_ethnicities_orcish_rework.txt

```diff
@@ -0,0 +1,388 @@
+orcish_rework = {
+	template = "caucasian"
+	
+	skin_color = {
+		10 = { 0.302 0.459 0.329 0.729 } # Green
+	}
+	eye_color = {
+        10 = { 0 0.337 0.259 0.678 } # Yellow, Orange, Red
+		10 = { 0.537 0.329 1 0.686 } # The rest
+	}
+	hair_color = {
+		10 = { 0.18 1 0.18 1 } # Black
+		10 = { 0 0.894 0.059 0.973 } # Brown
+		10 = { 0.796 0.822 0.900 0.937 } # Dark Blue, Purple
```

Comment:
Blue?

Reply:
> 
> 
> Blue?

From [here](https://wow.gamepedia.com/Orc#Appearance), under the Hairs heading:

> In World of Warcraft, orcs are shown with a variety of hair colors; **black, brown, blues, reds, purples and with age, grey and white.**

If it's too blue, I can tone it down.

Reply:
> If it's too blue, I can tone it down.

I wouldn't allow such colors. Don't remember orcish chieftains with purple or blue hairs.

Reply:
The same with eyes.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/107889974-93474680-6f2f-11eb-9847-b3be3c83e194.png)

</details>
In WoW they're added for the player customization but not used by lore characters.

Reply:
Changed in 715d603825fb5d5d7e0762758538ff3052d8fd63.